plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go

```diff
@@ -225,7 +225,8 @@ func defaultPriorities() sets.String {
 		// Prioritizes nodes that have labels matching NodeAffinity
 		factory.RegisterPriorityFunction2("NodeAffinityPriority", priorities.CalculateNodeAffinityPriorityMap, priorities.CalculateNodeAffinityPriorityReduce, 1),
 
-		// TODO: explain what it does.
+		// Taints are restrictive markers added to Nodes. Only Pods which explicitly tolerate a particular taint
+		// will be scheduled onto a Node marked with that taint.
```

Comment:
It's better to say: `// Prioritizes nodes that ...` and please limit this comment in one sentence and accurate.